### PR TITLE
Remove timestamp and global_slot_since_hard_fork in precondition

### PR DESCRIPTION
**Pull request by ghost-not-in-the-shell**
Explain your changes:
This PR removes `timestamp` and `global_slot_since_hard_fork` from zkapp_preconditions. Those changes would also affect snarkyjs repo. Once this PR is merged, the changes to snarkyjs would also be merged.

Explain how you tested your changes:
*

Checklist:

- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by mitschabaude — APPROVED** [review]
Nice change!

File: frontend/mina-signer/tests/party.test.ts [hunk 1]
```diff
@@ -48,7 +48,6 @@ let mockedZkappCommand = {
         preconditions: {
           network: {
             snarkedLedgerHash: null,
-            timestamp: null,
             blockchainLength: { lower: "0", upper: "4294967295" },
             minWindowDensity: { lower: "0", upper: "4294967295" },
             totalCurrency: { lower: "0", upper: "18446744073709551615" },
```
[the remark this answers is not in the document]
Comment (ghost-not-in-the-shell) [reply frontend/mina-signer/tests/party.test.ts:53]: I think I already removed that.

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by deepthiskumar — CHANGES_REQUESTED** [review]

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

File: frontend/mina-signer/tests/party.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mitschabaude) [reply frontend/mina-signer/tests/party.test.ts:50]: I meant in this test file!

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by deepthiskumar — APPROVED** [review]

**Pull request merged**